simple_app/templates/about.html

```diff
@@ -0,0 +1,59 @@
+{% extends "base.html" %}
+{% set active_page = "about" %}
+
+{% block content %}
+
+
+<div class='container'>
+    <h1 class="display-2">The SIMPLE Archive</h1>
+    <p class="display-6">
+        The vision of the SIMPLE project is to create a collaborative database of low-mass stars, brown dwarfs,
+        and directly imaged exoplanets: a simple archive of complex objects.
+        The tables and fields currently included in the database are described in the
+        <a href="https://github.com/SIMPLE-AstroDB/SIMPLE-db/blob/main/documentation/README.md" target="_blank">
+        Documentation</a> and currently include names, coordinates, photometry and reference and data
+        provenance information and is visualized in the
+        <a href="https://github.com/SIMPLE-AstroDB/SIMPLE-db#simple-database-schema" target="_blank"> schema</a>.
+    </p>
+    <p class="display-6">
+        We are currently working on including kinematics, spectra, images, and modelled and retrieved parameters.
+        We are developing several different methods to interact with the database, including python,
+        a website and API, and database browsers.
+        This database uses the <a href="https://docs.sqlalchemy.org/en/14/orm/index.html" target="_blank">
+        SQLAlchemy ORM</a> and is designed to be interacted with via the
+        <a href="https://pypi.org/project/astrodbkit2/" target="_blank"> astrodbkit2</a> package.
+        To see more details about how this project got started and our first discussions,
+        check out the <a href="https://github.com/SIMPLE-AstroDB/SIMPLE-db/wiki/Original-Notes" target="_blank">
+        archived running notes in the Wiki</a>.
+    </p>
+    <h2 class="display-3">Getting Involved</h2>
+    <p class="display-6">
+        If you'd like to take part or just stay in the loop as this project progresses,
+        please request to join the
+        <a href="https://groups.google.com/forum/#!forum/simple-archive" target="_blank"> discussion list</a>.
+        For day-to-day discussions, please join us in the <i>#simple-db-dev</i> channel in the Astropy Slack workspace.
+        If you are not already in the <a href="https://astropy.slack.com/" target="_blank"> Astropy Slack</a>,
+        request an <a href="http://joinslack.astropy.org/" target="_blank"> account</a>.
+    </p>
+    <h3 class="display-5">Imposter syndrome disclaimer:</h3>
+    <p class="display-6">
+        We want your help. No, really.
```

Comment:
Do you want to add explicit new lines (`<br>`, I think)? It otherwise comes out as a big long paragraph.